packages/core/helper-plugin/rollup.config.js

```diff
@@ -1,58 +1,69 @@
-import babel from 'rollup-plugin-babel';
-import resolve from 'rollup-plugin-node-resolve';
-import commonjs from 'rollup-plugin-commonjs';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import nodePolyfills from 'rollup-plugin-node-polyfills';
+import babel from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
 import svg from 'rollup-plugin-svg';
+import image from '@rollup/plugin-image';
 import postcss from 'rollup-plugin-postcss';
-import { terser } from 'rollup-plugin-terser';
-import replace from 'rollup-plugin-replace';
-import filesize from 'rollup-plugin-filesize';
-import pkg from './package.json';
+import packageJson from './package.json';
 
 export default {
   input: './lib/src/index.js',
   output: [
     {
-      exports: 'named',
-      file: pkg.main,
+      file: packageJson.main,
       format: 'cjs',
-      sourceMap: false,
-      name: '@strapi/helper-plugin',
-      compact: true,
-      globals: {
-        react: 'React',
-        'react-dom': 'ReactDOM',
-      },
+      sourcemap: true,
```

Comment:
I'm not familiar with the rollup config, why do we set the sourcemap value to true now?

Reply:
You areright it should be set to false!